src/Form/ConceptSchemeForm.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\rdf_skos\Form;
+
+use Drupal\Core\Entity\ContentEntityForm;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Form controller for SKOS Concept Scheme edit forms.
+ */
+class ConceptSchemeForm extends ContentEntityForm {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function save(array $form, FormStateInterface $form_state): void {
+    $entity = $this->entity;
+
+    $status = parent::save($form, $form_state);
+
+    switch ($status) {
```

Comment:
The switch statement is used for only one value, shouldn't it be better to do:

```php
$value = 'something';

if ($status === SAVED_NEW) {
  $value = 'something else';
}
```

Reply:
I think we will remove the forms...so not gonna spend time on this,.